### PR TITLE
Wizard

### PR DESCRIPTION
**Pull request by thure**
## [Spec in Figma ↗️](https://www.figma.com/file/EOsbapNvZgEwcA1mShswfh/Microsoft-Teams-UI-Kit-Community?node-id=388%3A8)

This PR implements the Wizard HVC. For this initial implementation, the developer is responsible for responding to user interactions that would cause a change in content, e.g. clicking 'Back' or clicking an item in the Wizard's sidebar.

Keyboard interactions are not aligned with the interactions specified in Figma at this time; I'm working with the interaction designer to confirm exceptions should be made to functionality delivered out-of-the-box by Fluent UI.

### CODE REVIEW

**Comment by akatrukhin** [comment]
Screen Shot 2021-02-09 at 1 33 51 PM - Gap between wizard and scroll looks weird to me. It will be great to have a scroll area inside of the Wizzard component.

File: stories/Wizard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      title: fake("{{company.catchPhrase}}"),
+      preface: fake("{{lorem.sentences}}"),
+    },
+    sections: [
+      {
+        title: fake("{{company.catchPhrase}}"),
+        preface: fake("{{lorem.sentence}}"),
+        inputGroups: [
+          {
+            type: "text-inputs" as "text-inputs",
```
Comment (akatrukhin) [inline stories/Wizard.stories.tsx:62]: "text-inputs" as "text-inputs", "text" as "text" and etc.
Could you create a Type or Enum for those valuse. Code will be much cleaner and IDE could show all available options for current property.

**Review by akatrukhin — CHANGES_REQUESTED** [review]
Lets wait for design review, I'm expecting style updates.

**Comment by akatrukhin** [comment]
Screen Shot 2021-02-10 at 11 34 29 AM
Screen Shot 2021-02-10 at 11 34 50 AM

At default theme, there is no hover state for passed step.
At dark theme, hover state is different

High Contrast doesn't show next steps at all
Screen Shot 2021-02-10 at 11 35 06 AM

High Contrast dropdown invisible also:
Screen Shot 2021-02-10 at 11 35 06 AM

For some reason on my end, I can't edit field after I insert value and comeback to it. 
Screen Shot 2021-02-10 at 11 36 20 AM

File: stories/Wizard.stories.tsx (hunk 1 above)
Comment (thure) [reply stories/Wizard.stories.tsx:62]: I will investigate… I've noticed in the TS ecosystem, this is not an unusual pattern, but I agree the enum pattern is better where we can offer it.

**Comment by thure** [comment]
Reverting to draft due to #54.

**Comment by thure** [comment]
@akatrukhin I’ve now either resolved or wasn’t able to duplicate all of the items in your previous comments.

**Review by akatrukhin — APPROVED** [review]
Looks good, good job!

**Review by asnow003 — APPROVED** [review]

**Pull request merged**